external-crates/move/crates/move-analyzer/prettier-move/tests/control-flow/return_expression.move

```diff
@@ -0,0 +1,55 @@
+// option:
+// print_width: 80
+
+module test::return_expression {
+    public fun folding() {
+        // this is a return expression
+        return very_very > very_very_long_if_condition + 100;
+        return very_very_long_if_condition > very_very_long_if_condition &&
```

Comment:
e add tests with more complicated return expressions, in particular parenthesized in a nested fashion